xcell/mappers/mapper_P18CMBK.py

```diff
@@ -94,17 +94,25 @@ def get_nl_coupled(self):
     def get_cl_fiducial(self):
         if self.cl_fid is None:
             ell = self.get_ell()
-            cl = self.noise[2] - self.noise[1]
+            noise = self._get_noise()
+            cl = noise[2] - noise[1]
             cl = interp1d(self.noise[0], cl, bounds_error=False,
```

Comment:
why use `self.noise` here but `noise` above? I know they're copies of the same, but it's a bit confusing.

Reply:
It should be `noise`. Fixed